src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1210,12 +1233,47 @@ void WholeBodyDynamicsDevice::filterSensorsAndRemoveSensorOffsets()
 
         iDynTree::Wrench rawFTMeasureWithOffsetRemoved  = ftProcessors[ft].filt(rawFTMeasure);
 
+
+        bool isNaN = false;
```

Comment:
I would remove this check, in the end once we removed the bugs from the processing of data, this should always be fine, and I don't think it make sense to have checks on all the points of the code.